src/test/integration/nodemailer.test.ts

```diff
@@ -0,0 +1,65 @@
+import enmail from '../../index';
+
+jest.mock('nodemailer');
+const nodemailer = require('nodemailer');
+nodemailer.createTransport.mockReturnValue({ sendMail: jest.fn(data => data) });
+
+describe('Nodemailer adapter', () => {
+    const getGmailAdapter = () => enmail.createAdapter.nodemailer(options);
+    const options = {
+        service: 'gmail',
+        auth: {
+            type: 'login' as any,
```

Comment:
Is the cast required? If so, I think we ought to fix that. Devs that want to use the library will peek into tests, if they don't find what they're looking for in the readme, so they'll just copy that.

Reply:
i changed the `as any` to `as const` it looks there is no enum which we can use